alternator/streams.cc

```diff
@@ -595,44 +592,39 @@ struct shard_iterator {
     static auto constexpr inclusive_marker = 'I';
     static auto constexpr exclusive_marker = 'i';
 
-    shard_id shard;
+    static auto constexpr uuid_str_length = 36;
```

Comment:
I don't have a better place to leave this comment so I'll randomly leave it hereI:
If the purpose of this patch is to fix the length of shard ids, which need to be shorter than some length, please add a test which verifies this length - which should fail before this patch and pass after (and of course also on DynamoDB). It doesn't need to be a whole new test - can be an additional assert in one of the many existing tests which already deal with shard ids (please with a comment explaining what is being tested).

Reply:
test assert added.